src/Stratis.Bitcoin.IntegrationTests/Wallet/SendingTransactionOverPolicyByteLimitSpecification_Steps.cs

```diff
@@ -28,8 +28,7 @@ public partial class SendingTransactionOverPolicyByteLimit : BddSpecification
         private int CoinBaseMaturity;
         private Exception caughtException;
         private const string WalletName = "mywallet";
-        private const string WalletPassword = "123456";
-        private const string WalletPassphrase = "passphrase";
+        private const string WalletPassword = "password";
```

Comment:
These won't be required if you change the GetUnusedAddres to be like I suggested in ApiSteps.

Reply:
Looks like this test will be a bit more complicated. Maybe we merge this PR and I continue on these in the next PR?

Reply:
sure